sklbench/benchmarks/sklearn_estimator.py

```diff
@@ -414,12 +423,28 @@ def measure_sklearn_estimator(
                         data_args = (x_train,)
                     else:
                         data_args = (x_test,)
-                batch_size = get_bench_case_value(
-                    bench_case, f"algorithm:batch_size:{stage}"
-                )
-                if batch_size is not None:
+
+                if method == "partial_fit":
+                    num_batches = get_bench_case_value(bench_case, "data:num_batches")
+                    batch_size = get_bench_case_value(bench_case, "data:batch_size")

```

Comment:
Instead of separate branch for `partial_fit`, extend mechanism of `online_inference_mode` to partial fitting too.

Reply:
could you provide the exact link to implementation of this mechanism? i was not able to find the usage of this parameter, just see its setting in the config.

Reply:
Actrually, `online_inference_mode` was removed as unnecessary before merge of refactor branch. This mode is enabled by `batch_size != None` only.

Reply:
You can split batch size into two for training and inference.

Reply:
> Actually, `online_inference_mode` was removed as unnecessary before merge of refactor branch.
what should I extend then?